projects/plugins/jetpack/extensions/blocks/contact-form/edit.js

```diff
@@ -207,15 +212,24 @@ export function JetpackContactFormEdit( {
 					icon={ get( blockType, [ 'icon', 'src' ] ) }
 					label={ get( blockType, [ 'title' ] ) }
 					instructions={ __(
-						"Please select which type of form you'd like to add, or create your own using the skip option.",
+						'Start building a form by selecting one of these form templates, or search in the patterns library for more forms:',
 						'jetpack'
 					) }
 					variations={ filter( variations, v => ! v.hiddenFromPicker ) }
-					allowSkip
 					onSelect={ ( nextVariation = defaultVariation ) => {
 						setVariation( nextVariation );
 					} }
 				/>
+				<Flex>
+					<Button className="form-placeholder__external-link" href={ CUSTOMIZING_FORMS_URL }>
```

Comment:
I think there's a `ExternalLink` component you can use

Reply:
https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/external-link/README.md

Reply:
Done! 🙂